### PR TITLE
Adding unit test cases.

### PR DESCRIPTION
**Pull request by anandmohan777**

### CODE REVIEW

**Review by AlexanderSenf — COMMENTED** [review]
The coverage looks good!
Would it be possible to use real keys in the test (e.g. as here https://github.com/uio-bmi/crypt4gh/blob/master/src/test/java/no/ifi/uio/crypt4gh/stream/Crypt4GHOutputStreamTest.java) and then to test the functions reading and extracting the key, etc.?

**Comment by anandmohan777** [comment]
@AlexanderSenf  Added test cases using real keys.

File: src/test/java/eu/elixir/ega/ebi/keyproviderservice/config/MyCipherConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    }
+
+    /**
+     * Test {@link MyCipherConfig#readFileAsString(String)}. Verify the
+     * pgpPrivateKey.
+     */
+    @Test
+    public void testExtractKey() throws IOException {
+        PGPPrivateKey pgpPrivateKey = myCipherConfig.extractKey(getPrivateKey(), getPassphrase());
+        assertNotNull(pgpPrivateKey);
```
Comment (juhtornr) [inline src/test/java/eu/elixir/ega/ebi/keyproviderservice/config/MyCipherConfigTest.java]: Is it possible to verify that extracted key is correct instead just checking that it isn't NULL?

**Comment by anandmohan777** [comment]
@juhtornr Yes, we can check the PGPPrivateKey keyID. So I have changed it and added this thing.

**Review by juhtornr — APPROVED** [review]
I reviewed this and new test are comprehensive and well written so I approve.

**Pull request merged**